src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/StationsAppSchemaTestSupport.java

```diff
@@ -0,0 +1,95 @@
+/* (c) 2018 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.AbstractMap.SimpleEntry;
+import java.util.Collections;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.custommonkey.xmlunit.XpathEngine;
+import org.geoserver.data.test.SystemTestData;
+import org.junit.Before;
+import org.w3c.dom.Document;
+
+/** Test support for GML 3.1 and GML 3.2 Stations data use case */
+public abstract class StationsAppSchemaTestSupport extends AbstractAppSchemaTestSupport {
+
+    protected final Map<String, String> GML31_PARAMETERS =
+            Collections.unmodifiableMap(
+                    Stream.of(
+                                    new SimpleEntry<>("GML_PREFIX", "gml31"),
+                                    new SimpleEntry<>(
+                                            "GML_NAMESPACE", "http://www.opengis.net/gml"),
+                                    new SimpleEntry<>(
+                                            "GML_LOCATION",
+                                            "http://schemas.opengis.net/gml/3.1.1/base/gml.xsd"))
+                            .collect(Collectors.toMap((e) -> e.getKey(), (e) -> e.getValue())));
+
```

Comment:
If I understood correctly basically this barely readable _patchwork_ of Java 8 code means:
```
Map<String, String> gml32Parameters = new HashMap<>();
gml32Parameters.put("GML_PREFIX", GML32_PREFIX);
gml32Parameters.put("GML_NAMESPACE", "http://www.opengis.net/gml/3.2");
gml32Parameters.put("GML_LOCATION", "http://schemas.opengis.net/gml/3.2.1/gml.xsd");
```

Reply:
You're right, in general conditions the good old Map building is the best way to do.
In this case this lines are refactored from the original subclass, and are class property one-statement map definitions (protected final Map<String, String> GML31_PARAMETERS = ... ; ) and seems to be useful.